src/integration/pages/onDemandAudioPage/crossPlatformTests.js

```diff
@@ -10,4 +13,7 @@ export default service => {
   runCommonEpisodeTests();
   runRecentEpisodesTests();
   runMediaPlayerEmbedLegacyTests();
+  runCoreCanonicalTests();
+  runCanonicalAnalyticsTests();
+  runRadioScheduleTests({ isAmp: false }); // keep this for now until live radio also has amp removed
```

Comment:
`isAmp` is false by default, so let's remove it

```suggestion
  runRadioScheduleTests();
```

Reply:
Also - not sure if I mentioned this already or not. I think we should move the crossPlatform tests to the canonical file - since onDemandAudio is no longer cross platform, this is confusing.

Reply:
I was thinking we would change the name of the crossPlatform test file anyway, as yes it doesn't mean anything anymore. But also the 'canonical' distinction doesn't mean anything now either, so I think even using the word canonical is maybe puzzling. In the case of me changing the file name, I don't think I need to change any of the other code as they have now been consolidated into one file, and we can just choose what to call that file. I may have missed something though that you have seen related to consolidating the files in the different direction.

Reply:
The tests fail if I remove this without providing the default false value, but you suggested reverting that in the comment https://github.com/bbc/simorgh/pull/11799#discussion_r1692955171, although I may have misunderstood the reasoning

Reply:
An optional param like `isAmp` should always have a default. So if it is defaulted to `false` then it means we don't need to include it. Hopefully reverting the removal of the default value will fix the test - worth a shot?